payloads/external/skiboot/Makefile

```diff
@@ -0,0 +1,36 @@
+## SPDX-License-Identifier: GPL-2.0-only
+
+skiboot_dir=$(CURDIR)/talos-skiboot
+skiboot_git_repo=https://git.raptorcs.com/git/talos-skiboot
+skiboot_revision=9858186353f2203fe477f316964e03609d12fd1d
+dts=talos_ii_device-tree_stripped.dts
+skiboot=skiboot.lid
+uimage=skiboot.uimage
+config=config.its
+
+unexport $(COREBOOT_EXPORTS)
+
+.PHONY: all clean distclean
+
+all: $(CURDIR)/$(uimage)
+
+$(CURDIR)/$(uimage): $(skiboot_dir)/$(skiboot) $(CURDIR)/fdt.bin
+	cp $(skiboot_dir)/$(skiboot) $(CURDIR)/$(skiboot)
```

Comment:
Do we have to copy this file or can we just add a path in config.its?

Reply:
We don't have to copy it. I have added a path to `config.its` and got rid of copying